apps/src/lab2/views/dialogs/types.ts

```diff
@@ -6,6 +10,7 @@ import {GenericPromptProps} from './GenericPrompt';
 import {SkipDialogProps} from './SkipDialog';
 import {StartOverDialogProps} from './StartOverDialog';
 
+export type SubmissionStatusType = ValueOf<typeof ProjectSubmissionStatus>;
```

Comment:
Hmm technically speaking this is isn't a lab2-specific type (though I know we're only implementing in lab2 for now). Maybe this belongs closer to the API itself since that's what it's used for?